docs/config_file.md

```diff
@@ -0,0 +1,14 @@
+# Kops config file
```

Comment:
Can we create a FAQ or another doc or something like https://kubernetes.io/docs/user-guide/kubectl-cheatsheet/

We have documentation sprawl and I am trying to figure out how to tighten up and make the documentation more accessible.  I am open to this document if we can include this section in a larger document as well.  Since I am a maintainer I am the wrong person to brainstorm and have an opinion.  What do you think?  @kenden thoughts?